external/skiboot/libstb/secvar/crypto/crypto-mbedtls-v3.c

```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: Apache-2.0
+/* Copyright 2021 IBM Corp.*/
+#ifdef MBEDTLS_V3
+// ^extra precaution to not compile with mbedtls version 3.x unless specified
```

Comment:
This isn't necessary, and the compilation should fail if somehow both 2.x and 3.x are selected, OR compiling with 3.x is attempted without actually having 3.x